src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -2804,6 +2816,27 @@ FixedwingPositionControl::initializeAutoLanding(const hrt_abstime &now, const po
 	}
 }
 
+bool FixedwingPositionControl::checkLowHeightConditions()
+{
+	// Are conditions for low-height
+	return _param_fw_t_thr_low_hgt.get() >= 0.f && _local_pos.dist_bottom_valid
+	       && _local_pos.dist_bottom < _param_fw_t_thr_low_hgt.get();
```

Comment:
I am not sure the way this is handled is desirable. This would change the TECS constant when the vehicle is flying lower than the launch altitude regardless of whether it is trying to land or navigating.

Reply:
@Jaeyoung-Lim Maybe you've missed it, but this is not local altitude but distance to the ground. So this is only available when the ekf has a distance measurement to the ground. So it's completely decoupled from home altitude.